src/test/resources/reporter_buildOpenReacParameters_caseRelativeTrue.json

```diff
@@ -0,0 +1,152 @@
+                "type": "VOLTAGE"
+              }
+            }
+          },
+          {
+            "reportKey": "voltageLimitModified",
+            "values": {
+              "initialLowVoltageLimit": {
+                "value": "NaN",
+                "type": "VOLTAGE"
+              },
+              "voltageLevelId": {
+                "value": "VLHV1",
+                "type": "VOLTAGE_LEVEL"
+              },
+              "reportSeverity": {
+                "value": "TRACE",
+                "type": "SEVERITY"
+              },
+              "newHighVoltageLimit": {
+                "value": 10.0,
+                "type": "VOLTAGE"
+              },
+              "initialHighVoltage": {
+                "value": 20.0,
+                "type": "VOLTAGE"
+              },
+              "newLowVoltageLimit": {
+                "value": "NaN",
+                "type": "VOLTAGE"
+              }
+            }
+          },
+          {
+            "reportKey": "voltageLimitModified",
+            "values": {
+              "initialLowVoltageLimit": {
+                "value": 10.0,
+                "type": "VOLTAGE"
+              },
+              "voltageLevelId": {
+                "value": "VLGEN",
+                "type": "VOLTAGE_LEVEL"
+              },
+              "reportSeverity": {
+                "value": "TRACE",
+                "type": "SEVERITY"
+              },
+              "newHighVoltageLimit": {
+                "value": 10.0,
+                "type": "VOLTAGE"
+              },
+              "initialHighVoltage": {
+                "value": 20.0,
+                "type": "VOLTAGE"
+              },
+              "newLowVoltageLimit": {
+                "value": -10.0,
+                "type": "VOLTAGE"
+              }
+            }
+          },
+          {
+            "reportKey": "missingVoltageLimits",
+            "values": {
+              "nbMissingVoltageLimits": {
+                "value": 0
+              },
+              "reportSeverity": {
+                "value": "INFO",
+                "type": "SEVERITY"
+              }
+            }
+          },
+          {
+            "reportKey": "voltageLimitModifications",
+            "values": {
+              "reportSeverity": {
+                "value": "INFO",
+                "type": "SEVERITY"
+              },
+              "nbVoltageLimitModifications": {
+                "value": 0
+              }
+            }
+          }
+        ]
+      }
+    ]
+  },
+  "dics": {
+    "default": {
+      "restrictedVoltageLevels": "The modifications to the low limits for certain voltage levels have been restricted to avoid negative voltage limits: ${joinedVoltageLevelsIds}",
+      "voltageLimitModifications": "Voltage limits of ${nbVoltageLimitModifications} voltage levels have been modified according to user input.",
+      "VoltageInit": "VoltageInit",
+      "OpenReactParameters": "OpenReact parameters",
```

Comment:
It's **OpenReac**, this typo is also elsewhere: to fix

Reply:
oh, thanks

Reply:
huh, I think there is a typo problem in powsybl module also
```java
import com.powsybl.openreac.parameters.input.json.OpenReactJsonModule;
// [...]
objectMapper.registerModule(new OpenReactJsonModule());
```